src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/TypeOperations.cs

```diff
@@ -2460,7 +2460,7 @@ protected internal override Expression VisitLambda<T>(Expression<T> node)
                 {
                     return node;
                 }
-                return node.Update(b, node.Parameters);
+                return Expression.Lambda<T>(b, node.Name, node.TailCall, node.Parameters);
```

Comment:
I am not sure why this changed. I saw similar changes in couple changes and assumed that at some point static generic methods either did not work with AOT or were very slow. Same with virtual generic methods. I think they are working fine now as long as they do not lead to infinitary expansions (and even that might be handled by now,... not sure)